game-core/src/main/java/games/strategy/triplea/ui/ObjectiveProperties.java

```diff
@@ -0,0 +1,40 @@
+package games.strategy.triplea.ui;
+
+import java.time.Instant;
+import java.util.Map.Entry;
+import java.util.Set;
+
+/**
+ * Loads objective text from objectives.properties.
+ */
+public class ObjectiveProperties extends PropertyFile {
+
+  private static final String PROPERTY_FILE = "objectives.properties";
+  private static final String OBJECTIVES_PANEL_NAME = "Objectives.Panel.Name";
+  static final String GROUP_PROPERTY = "TABLEGROUP";
+
+
+  private static ObjectiveProperties instance = null;
+  private static Instant timestamp = Instant.EPOCH;
+
+  protected ObjectiveProperties() {
+    super(PROPERTY_FILE);
+  }
+
+  public static ObjectiveProperties getInstance() {
+    // cache properties for 1 second
+    if (instance == null || timestamp.plusSeconds(1).isBefore(Instant.now())) {
+      instance = new ObjectiveProperties();
```

Comment:
+1 for unifying those objects into a single class.
Maybe the constructor could pass a supplier for a new instance as well, so this code wouldn't be duplicated either.